src/applications/disability-benefits/all-claims/content/form0781/workflowChoicePage.jsx

```diff
@@ -0,0 +1,159 @@
+import React from 'react';
+
+export const workflowChoicePageTitle = (
+  <div className="vads-u-font-size--h3">
+    Statement about mental health conditions (VA Form 21-0781)
+  </div>
+);
+
+// Lists new conditions the veteran has claimed
+const conditionSelections = formData => {
+  const conditions = formData?.newDisabilities?.map(
+    disability =>
+      // Capitalize condition
+      disability.condition.charAt(0).toUpperCase() +
+      disability.condition.slice(1),
+  );
+
+  if (conditions?.length) {
+    return (
+      <div>
+        <p>You selected these new conditions for your disability claim:</p>
+        <ul>
+          {conditions.map((condition, index) => (
+            <li key={index}>{condition}</li>
+          ))}
+        </ul>
+      </div>
+    );
+  }
+
+  return <></>;
```

Comment:
Shouldn't there always be at least 1 condition for this page to render?

Reply:
Yes, this is pretty defensive, noted this [here](https://github.com/department-of-veterans-affairs/vets-website/pull/33655/files/320f881724502c90482556dc04628e209e440c6e#diff-5485bdc8546bee34e56df206822fac64a4da9aff823504a10efc54a53f6c0323R97) in the test.

I would assume we would want this page to render with the other helpful information instead of blowing up if there is a bug that causes the conditions to be missing and still allow getting to this page. What do you think? Maybe we can ask Ruben and Lisa about this

Reply:
I did notice your inclusion of that test, but was also wondering if the page shouldn't be rendered at all. Not sure what protocol is in this situation.

Reply:
Yeah revisiting this and the [content that follows it ](https://github.com/department-of-veterans-affairs/vets-website/pull/33655/files#diff-069586937c2b741226c820e0f34bc68962a1ed01a5194b71bfde4d334bdfa2ccR39) I think the page copy would never make a whole lot of sense without the conditions and we would want it to blow up. 

So I think I am going to remove the guard and the associated test here

Reply:
Did this in https://github.com/department-of-veterans-affairs/vets-website/pull/33655/commits/9495de49b142380baff48cbff69e3e22e724e0e8